bundle-workflow/src/run_integ_test.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+import os
+import subprocess
+import sys
+
+from git.git_repository import GitRepository
+from manifests.build_manifest import BuildManifest
+from manifests.bundle_manifest import BundleManifest
+from manifests.test_manifest import TestManifest
+from system.temporary_directory import TemporaryDirectory
+from test_workflow.integ_test_suite import IntegTestSuite
+
+# TODO: 1. log test related logging into a log file. Output only workflow logs on shell.
+# TODO: 2. Move common functions to utils.py
+
+
+COMMON_DEPENDENCIES = [
+    'OpenSearch',
+    'common-utils',
+    'job-scheduler',
+    'alerting'
+]
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
+    parser.add_argument('--bundle-manifest', type=argparse.FileType('r'), help="Bundle Manifest file.")
+    parser.add_argument('--build-manifest', type=argparse.FileType('r'), help="Build Manifest file.")
+    parser.add_argument('--test-manifest', type=argparse.FileType('r'), help="Test Manifest file.")
+    parser.add_argument('--keep', dest='keep', action='store_true',
+                        help="Do not delete the working temporary directory.")
+    args = parser.parse_args()
+    return args
+
+
+# TODO: replace with DependencyProvider - https://github.com/opensearch-project/opensearch-build/issues/283
+def pull_common_dependencies(work_dir, build_manifest):
+    print("Pulling common dependencies for integration tests")
+    print("Pulling opensearch-build")
+    os.chdir(work_dir)
+    GitRepository('https://github.com/opensearch-project/opensearch-build.git', 'main',
+                  os.path.join(work_dir, 'opensearch-build'))
+    for component in build_manifest.components:
+        if component.name in COMMON_DEPENDENCIES:
+            print("Pulling " + component.name)
+            GitRepository(component.repository, component.commit_id, os.path.join(work_dir, component.name))
+
+
+# TODO: replace with DependencyProvider - https://github.com/opensearch-project/opensearch-build/issues/283
+def sync_dependencies_to_maven_local(work_dir, manifest_build_ver):
+    os.chdir(work_dir + "/OpenSearch")
+    subprocess.run(work_dir + '/opensearch-build/tools/standard-test/integtest_dependencies_opensearch.sh opensearch ' + manifest_build_ver,
```

Comment:
We can pull the command as a common variable and then append `opensearch ` and `common-utils` to it. Just to make the change and avoid mirroring

Reply:
ack. this logic will go away very soon with s3 provided dependencies. just a stop gap for now.